pslab/bus/busio.py

```diff
@@ -373,3 +381,208 @@ def write_readinto(
 
         for i, v in zip(range(in_start, in_end), data):
             buffer_in[i] = v
+
+
+class Parity(Enum):
+    EVEN = 1
+    ODD = 2
+
+
+class UART(_UARTPrimitive):
+    """Busio UART Class for CircuitPython Compatibility.
+
+    Parameters
+    ----------
+    device : :class:`SerialHandler`, optional
+        Serial connection to PSLab device. If not provided, a new one will be
+        created.
+    baudrate : int, optional
+        The transmit and receive speed. Defaults to 9600.
+    bits : int, optional
+        The number of bits per byte, 8 or 9. Defaults to 8 bits.
+    parity : :class:`Parity`, optional
+        The parity used for error checking. Defaults to None.
+        Only 8 bits per byte supports parity.
+    stop : int, optional
+        The number of stop bits, 1 or 2. Defaults to 1.
```

Comment:
Here, `stop=0` means a single stop bit, and `stop=1` means two stop bits, correct? I think the value of `stop` should be the actual number of stop bits, i.e. 1 or 2, not 0 or 1.